src/term/mod.rs

```diff
@@ -345,6 +345,8 @@ pub struct Term {
 
     pub dirty: bool,
 
+    pub rang_bell: bool,
```

Comment:
I _think_ this flag will become obsolete once the VisualBell is moved onto this type.